docs/HOWTO/90_Misc/Unpack_dvd.rst

```diff
@@ -0,0 +1,105 @@
+.. _howto-unpack-dvd:
+
+HOWTO Use unpack_dvd to Extract the Full OS RPM Package Set
+===========================================================
+
+The SIMP ISO provides a minimal set of packages.
+
+If you require additional OS packages, you can extract them from vendor ISOs using
+:program:`unpack_dvd`.  Additionally you can extract the extract the PXE boot
+TFTP boot files from the ISO to the rsync directories.
+
+Use option :code:`unpack_dvd --help` to see all options available for :program:`unpack_dvd`.
+
+Extract the OS Packages
+-----------------------
+
+:program:`Unpack_dvd` extracts the OS RPMs to :file:`/var/www/yum/<OperatingSystem>`
+in a directory named after the OS version it determines from files on the ISO.
+It then creates a link to the major version.  If version 7.6 is extracted,
+7 will be linked to 7.6.
+
+After extracting all the RPMs it will create or update a repository in the
+versioned directory.
+
+.. NOTE::
+
+   If unpack_dvd can only determine the major OS version from the files
+   on the ISO, :program:`unpack_dvd` will ask you to supply a more descriptive
+   version number using ``-v`` option.
+
+The following example will:
+
+* extract the RPMs to :file:`/var/www/yum/CentOS/7.8.2003`
+* create a repository in the above directory
+* link :file:`/var/www/yum/CentOS/7` to the above directory.
+
+#. Log on as ``simp`` and run :command:`sudo su - root`.
+#. Copy the appropriate vendor OS ISO(s) to the server.
+#. If the server where you are unpacking the vendor ISO was **NOT** built using the SIMP ISO ,
+   you must create :file:`/var/www/yum` (or the directory you indicated in ``-d``
+   option.)
+#. unpack using :program:`unpack_dvd`
+
+   .. code:: bash
+
+      # unpack_dvd -v <os version> <full path to iso>
+      unpack_dvd -v 7.8.2003 /myisodir/CentOS-7-x86_64-DVD-2003.iso
+
+#. Ensure that subsequent :term:`yum` operations are aware of the new RPM
+   packages by refreshing the system's yum cache:
+
+   .. code:: bash
+
+      yum clean all && yum makecache
+
+.. WARNING::
+
+   At this time :program:`unpack_dvd` does not work entirely with EL8 ISOs.
+   EL8 introduced modules to repositories and :program:`unpack_dvd` can not handle these.
+   It will extract some but not all of RPMs on an EL8 ISO. The files extracted
+   are enough to kickstart a basic EL8 system but some of the application stream
+   files are not extracted and added to the repository correctly.
+
+Extract PXE files
+-----------------
+
+Extracting the PXE files along with the OS files, was added to :program:`unpack_dvd` in :package:`simp-utils-6.4.0`.
+
+By default :program:`unpack_dvd` will pull information off the ISO and, using this information, create a directory named <os-family>-<version>-<arch>  under the tftpboot rsync directory and extract the PXE files there.
```

Comment:
Suggested replacment:
When PXE boot file extraction is enabled with the :command:`-X` option,
:program:`unpack_dvd` will, by default,

* extract OS version information from the ISO metadata
* create a directory for the PXE boot files in the rsync directory used by the
  :pupmod:`simp/tftpboot` module for the `production` Puppet environment

  * The new directory will be named :file:`<os-family>-<version>-<arch>`.
  * The `production` rsync directory is located at
    :file:`/var/simp/environments/production/rsync/<os family>/Global/tftpboot/linux-install/`.
  * See :ref:`Configure_PXE_Boot` for more information about using
    :pupmod:`simp/tftpboot`.

* extract the PXE files to the created OS directory.